pkg/proxy/iptables/proxier.go

```diff
@@ -668,8 +668,12 @@ func (proxier *Proxier) OnNodeAdd(node *v1.Node) {
 	}
 
 	proxier.mu.Lock()
-	proxier.nodeLabels = node.Labels
+	proxier.nodeLabels = map[string]string{}
```

Comment:
https://play.golang.org/p/kjY-IPixXlv

Reply:
I may be wrong though, it will be good if someone with more golang insights double check, but my understanding is that you pass a pointer to the map on node

Reply:
So you think this is a bug? Is that something report or failed info to support it?
And on the other hand, if that, you should also change some in ipvs code.
https://github.com/kubernetes/kubernetes/blob/b0abe89ae259d5e891887414cb0e5f81c969c697/pkg/proxy/ipvs/proxier.go#L979

https://github.com/kubernetes/kubernetes/blob/b0abe89ae259d5e891887414cb0e5f81c969c697/pkg/proxy/ipvs/proxier.go#L998

Reply:
I think that is safer to copy , I need to change ipvs too, thanks for catching it